pkg/pb/metainfo.proto

```diff
@@ -24,6 +31,65 @@ service Metainfo {
     rpc ProjectInfo(ProjectInfoRequest) returns (ProjectInfoResponse);
 }
 
+message BucketItem {
```

Comment:
there is already a `Bucket` so I changed the namee here. Re: https://github.com/storj/storj/blob/master/pkg/pb/inspector.proto#L116